src/styles/form-controls/_button.scss

```diff
@@ -78,20 +78,20 @@
 .alpheios-button-tertiary, button.alpheios-button-tertiary {
   @extend %alpheios-button;
   color: var(--alpheios-btn-tertiary-font-color);
-  background-color: var(--alpheios-btn-tertiary-bg-color);
+  background: var(--alpheios-btn-tertiary-bg-color);
```

Comment:
Do we change it to `background` because that's what the site is using and we want to override it? Background is a shorthand so by specifying it instead of `background-color` we're also specifying other background props implicitly (i.e. setting props such as `position` to their default values, I believe). So with `background` we're less specific than with `background-color`. I would think being more specific is better: it's more precise. Is using `background` justified in this case?

Reply:
I have changed it to the more general property because of two following reasons:
  - in the current situation (for the problem site)  `background-color` couldn't override `background`
  - these styles are defined for a specific type of button - that has only color by styleguide (no images, no other effects), so there is no need to define here only color; if someone would like to define another style (may be injherited from it) - it would anyway define all the properties.
     So from my point of view there is no sense to define only color here.

@kirlat , what do you think?

Reply:
Makes sense to me! Just wanted to understand the reasons behind that decision.